_inc/client/security/backups-scan.jsx

```diff
@@ -88,7 +88,7 @@ class BackupsScanRewind extends Component {
 						'You need to enter your server credentials to finish configuring Backups and Scan.',
 						'jetpack'
 					),
-					url: getRedirectUrl( 'calypso-settings-security', { site: siteRawUrl } ),
+					url: getRedirectUrl( 'jetpack-settings-security-credentials', { site: siteRawUrl } ),
```

Comment:
Shouldn't this one be named `calypso-settings-jetpack`, to better match where it lands?

Reply:
There was some discussion in Slack last week. I believe the outcome was for us to use keys what reference where a link points to, not where it has come from. cc @leogermani

Reply:
Right. Since that links goes to Calypso, shouldn't it stay prefixed with Calypso then, and updated to `jetpack` and instead of `security` since it now leads to the new Jetpack tab in Calypso settings.

Reply:
I think we meant to name redirects by their source, not their destination as we do now. The problem with using destinations is that we end up using the same redirect for multiple purposes (pieces of UI, routes, etc.), and then when we one to change one of the pieces of UI that point to that destination, we can't do it without changing the rest. 

This is exactly what is happening with `calypso-settings-security`. We can't change its destination to `/settings/jetpack/` because we still want it to point to `/settings/security` in some cases.

Reply:
That would make sense then 👍

I'll let @leogermani weigh in, he's the Redirect expert :)

Reply:
Yes, the idea is that the slug represents where the link is, and not where it points to. I've updated the Fielg Guide last week to reflect that.

This vision wasn't clear before and there are many links in Jetpack that don't follow this pattern. But I think it's a good idea to start doing it right now